packages/server/postgres/queries/insertStripeQuantityMismatchLogging.ts

```diff
@@ -0,0 +1,35 @@
+import getPg from '../getPg'
+import OrganizationUser from '../../database/types/OrganizationUser'
+import {insertStripeQuantityMismatchLoggingQuery} from './generated/insertStripeQuantityMismatchLoggingQuery'
+
+const insertStripeQuantityMismatchLogging = async (
+  userId: string,
+  eventTime: Date,
+  eventType: string,
+  stripePreviousQuantity: number,
+  stripeNextQuantity: number,
+  orgUsers: OrganizationUser[]
+) => {
+  return insertStripeQuantityMismatchLoggingQuery.run(
+    {
+      userId,
+      eventTime,
+      eventType,
+      stripePreviousQuantity,
+      stripeNextQuantity,
+      orgUsers: orgUsers.map(({id, inactive, joinedAt, removedAt, userId, role, tier}) => {
+        return {
+          id,
+          inactive,
+          joinedAt: JSON.stringify(joinedAt),
+          removedAt: removedAt ? JSON.stringify(removedAt) : "",
```

Comment:
nope, JSON.stringify will turn the Date object to a string by calling `.toJSON` and THEN turn that string into JSON, which means it'll give you an extra set of quotes

Reply:
Oh! I did not know these nuances and understand you wrongly before! Thanks for pointing it out!

Reply:
javascript is a horrible, strange language